### PR TITLE
Goodies

### PR DESCRIPTION
**Pull request by emanuele45**
Number of current notifications showed in the favicon and desktop notifications. :roller_coaster: :rocket: 

Plus some bug fixes as usual. :innocent:

### CODE REVIEW

File: sources/controllers/Mentions.controller.php [hunk 1]
```diff
@@ -171,6 +171,12 @@ public function pre_dispatch()
 	 */
 	public function action_index()
 	{
+		$req = HttpReq::instance();
+		if (isset($req->query->sa) && $req->query->sa == 'fetch')
```
Comment (Spuds) [inline sources/controllers/Mentions.controller.php]: I think (need to look at the code again :innocent: been lazy ) can be done as 
``` if ($req->getQuery('sa') === 'fetch') ```

Comment (emanuele45) [inline sources/controllers/Mentions.controller.php]: Ohhh... that's :cool: !! :sunglasses:

**Comment by emanuele45** [comment]
You didn't give me time to replace the getQuery! :P 
Ohh well, will fix later. :smile_cat:

**Comment by Spuds** [comment]
Well it was fine as it was :smile: it was just a reminder about getQuery and its other getBuddies  (which can also set default values and do data sanitation)

**Pull request merged**